scripts/inc.updatebinary.sh

```diff
@@ -89,7 +86,7 @@ contains() {
 }
 
 clean_inst() {
-  if [ -f /data/system/packages.xml ]; then
+  if [ -f /data/system/packages.xml ] && [ "$enforce_clean" != "true" ]; then
```

Comment:
Maybe rename to "forceclean", then it will coincide with the gapps-config keyword we would like to use.